terraform/projects/app-ecs-instances/main.tf

```diff
@@ -5,6 +5,12 @@
 *
 */
 
+variable "account_id" {
+  type        = "string"
+  description = "Our AWS account ID"
+  default     = "0"
```

Comment:
Could we document in the readme that people should be setting the account idea in their tfvars file so we can spin this up please? Actually, passing it in to the command line is probably better given it removes the risk that people commit them in the staging and production tfvars.

I kind of dislike the fact that we manually need to know the account ID to run any of the terraform. Might be worth double checking if there is any way we could get around this though...

Reply:
Maybe the terraform-pass integration so we could get it from reng-pass could be an option...

Reply:
did some spiking and it looks like we can use `data "aws_caller_identity"`: https://www.terraform.io/docs/providers/aws/d/caller_identity.html